webapp-client/src/component/CategoryList.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+// TODO: Add support to highlight active category.
+export default class CategoryList extends React.Component {
+  // this.props.categories: a list of category ids fetched from db
+  // this.props.setContentPane: sets the current category to be displayed
+  render() {
+    return (
+      <React.Fragment>
+        {this.props.categories.map((category, index) => {
+          return (
+            <a className="panel-block is-active" key={index} onClick={() => this.props.setContentPane(category)}>
```

Comment:
[React documentation](https://reactjs.org/docs/handling-events.html) recommends binding in the constructor. Also it is best to maintain consistency across your web app. Based on the other examples in the documentation, it looks like you should just wrap the props method inside a bound component method.